playground/keycloak/weathershop-realm.json

```diff
@@ -0,0 +1,2350 @@
+{
```

Comment:
Is all this necessary? Seems like a lot of these things should just be the default.

Reply:
That file is the result of an export performed on Keycloak itself. Can't control most of what gets exported there.

I do not feel like playing around with the file contents to see what is absolutely necessary.

Reply:
As @julioct says this is what Keycloak gives you when you export a realm.